lib/modules/versioning/ubuntu/index.spec.ts

```diff
@@ -195,9 +196,17 @@ describe('modules/versioning/ubuntu/index', () => {
     ${'groovy'}   | ${false}
     ${'hirsute'}  | ${false}
     ${'impish'}   | ${false}
-    ${'jammy'}    | ${false}
+    ${'jammy'}    | ${true}
   `('isStable("$version") === $expected', ({ version, expected }) => {
-    expect(ubuntu.isStable(version)).toBe(expected);
+    const orgNow = Settings.now;
+    const dt = DateTime.fromISO('2022-04-30');
+
+    try {
+      Settings.now = () => dt.valueOf();
```

Comment:
i think a better solution is to use jest.spyOn
```suggestion
      jest.spyOn(Settings, 'now').mockReturnValueOnce(dt.valueOf());
```

And do `jest.resetAllMocks` on `afterEach`